spec.md

```diff
@@ -2246,31 +2857,34 @@ Force on a non-passive bundle drives in the direction of each field's orientatio
 Example:
 
 ``` firrtl
-module Top:
-  input x : {a: UInt<2>, flip b: UInt<2>}
-  output y : {a: UInt<2>, flip b: UInt<2>}
+FIRRTL version 4.0.0
+circuit Top:
+  module Top:
+    input x : {a: UInt<2>, flip b: UInt<2>}
+    output y : {a: UInt<2>, flip b: UInt<2>}
 
-  inst d of DUT
-  connect d.x, x
-  connect y, d.y
+    inst d of DUT
+    connect d.x, x
+    connect y, d.y
 
-  wire val : {a: UInt<2>, b: UInt<2>}
-  connect val.a, UInt<2>(1)
-  connect val.b, UInt<2>(2)
+    wire val : {a: UInt<2>, b: UInt<2>}
+    connect val.a, UInt<2>(1)
+    connect val.b, UInt<2>(2)
 
-  ; Force takes a RWProbe and overrides the target with 'val'.
-  force_initial(d.xp, val)
+    ; Force takes a RWProbe and overrides the target with 'val'.
+    force_initial(d.xp, val)
 
-module DUT :
-  input x : {a: UInt<2>, flip b: UInt<2>}
-  output y : {a: UInt<2>, flip b: UInt<2>}
-  output xp : RWProbe<{a: UInt<2>, b: UInt<2>}>
+  module DUT :
+    input x : {a: UInt<2>, flip b: UInt<2>}
+    output y : {a: UInt<2>, flip b: UInt<2>}
+    output xp : RWProbe<{a: UInt<2>, b: UInt<2>}>
 
-  ; Force drives p.a, p.b, y.a, and x.b, but not y.b and x.a
-  wire p : {a: UInt<2>, flip b: UInt<2>}
-  define xp = rwprobe(p)
-  connect p, x
-  connect y, p
+    ; Force drives p.a, p.b, y.a, and x.b, but not y.b and x.a
+    wire p : {a: UInt<2>, flip b: UInt<2>}
+    define xp = rwprobe(p)
+    connect p, x
+    connect y, p
+  ;; snippetend
```

Comment:
Is the snippetbegin missing here?

Reply:
Yes! Good catch. Adding this in a force push.